packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -74,7 +74,7 @@ export class VaPagination {
     this.pageSelect.emit({ page });
 
     // Reset focus to the active page button.
-    (this.el.shadowRoot.activeElement as HTMLElement).blur();
+    (this.el?.shadowRoot?.activeElement as HTMLElement)?.blur();
```

Comment:
This line was [originally added](https://github.com/department-of-veterans-affairs/component-library/pull/602) because we had a defect where the selected page was not losing its active style (making it appears like two pages were selected at the same time).

![Screenshot 2023-08-08 at 2 27 24 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/4afe20a3-5c93-456f-b3e1-aa0b9089ed98)